### PR TITLE
Move folders around

### PR DESCRIPTION
**Pull request by levnikan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,13 +5,13 @@
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-  <title>JavaScript Fundamentals – Part 1</title>
+  <title>JavaScript Fundamentals</title>
```
Comment (levnikan) [inline index.html:8]: Why did you remove here?

**Review by levnikan — COMMENTED** [review]
All good!

**Review by Ivan-Andr — APPROVED** [review]
